custom_components/smart_envi/config_flow.py

```
@@ -0,0 +1,676 @@
+                    try:
+                        schedule_list = await client.get_schedule_list()
+                        for schedule in schedule_list:
+                            if isinstance(schedule, dict) and schedule.get("id") == schedule_id:
+                                self._schedule_data.update({
+                                    "enabled": schedule.get("enabled", self._schedule_data["enabled"]),
+                                    "name": schedule.get("name") or self._schedule_data["name"],
+                                    "temperature": schedule.get("temperature") or self._schedule_data["temperature"],
+                                    "times": schedule.get("times", self._schedule_data["times"]),
+                                })
+                                break
+                    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)